In-house patch to remove amqplib from Nova's requirements files as an alternate
implementation is used on Solaris.  This patch is Solaris-specific and
not suitable for upstream.

--- nova-2013.1.4/tools/pip-requires.orig       2013-11-22 08:08:07.506739321 -0700
+++ nova-2013.1.4/tools/pip-requires        2013-11-22 08:08:13.543791383 -0700
@@ -1,6 +1,5 @@
 SQLAlchemy>=0.7.8,<0.7.99
 Cheetah>=2.4.4
-amqplib>=0.6.1
 anyjson>=0.2.4
 argparse
 boto

--- nova-2013.1.4/nova.egg-info/requires.txt.orig       2013-11-22 08:09:22.863753329 -0700
+++ nova-2013.1.4/nova.egg-info/requires.txt        2013-11-22 08:09:28.573969576 -0700
@@ -1,6 +1,5 @@
 SQLAlchemy>=0.7.8,<0.7.99
 Cheetah>=2.4.4
-amqplib>=0.6.1
 anyjson>=0.2.4
 boto
 eventlet>=0.9.17
